### ISSUE TITLE
[Bug]: search data ,Memory error

### ISSUE THREAD

**Issue by Jfxhuangdao** [milvus-io/milvus#16805]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.0.2
- Deployment mode(standalone or cluster):standalone 
- SDK version(e.g. pymilvus v2.0.0rc2):2.0.2
- OS(Ubuntu or CentOS): Ubuntu 
- CPU/Memory: 1024G
- GPU: 
- Others:
```

### Current Behavior

6000W图片，512特征度，查询没问题。
1024特征图的查询异常。

### Milvus Log

```markdown
[2022/05/06 08:53:33.564 +00:00] [WARN] [rocksmq_impl.go:581] ["rocksmq produce too slowly"] [topic=by-dev-datacoord-timetick-channel] ["get lock elapse"=203] ["alloc elapse"=0] ["write elapse"=0] ["updatePage elapse"=0] ["produce total elapse"=203]
[2022/05/06 08:53:33.567 +00:00] [WARN] [rocksmq_impl.go:581] ["rocksmq produce too slowly"] [topic=by-dev-datacoord-timetick-channel] ["get lock elapse"=270] ["alloc elapse"=0] ["write elapse"=0] ["updatePage elapse"=0] ["produce total elapse"=270]
[2022/05/06 08:53:33.593 +00:00] [WARN] [rocksmq_impl.go:581] ["rocksmq produce too slowly"] [topic=by-dev-datacoord-timetick-channel] ["get lock elapse"=245] ["alloc elapse"=0] ["write elapse"=0] ["updatePage elapse"=0] ["produce total elapse"=245]
[2022/05/06 08:53:33.595 +00:00] [WARN] [rocksmq_impl.go:581] ["rocksmq produce too slowly"] [topic=by-dev-datacoord-timetick-channel] ["get lock elapse"=209] ["alloc elapse"=0] ["write elapse"=0] ["updatePage elapse"=0] ["produce total elapse"=209]
2022-05-06 08:53:33,653 DEBUG [default] [user@8d0e4a8dabd3] [void ReduceResultData(std::vector<milvus::SearchResult*>&, int64_t, int64_t)] [/go/src/github.com/milvus-io/milvus/internal/core/src/segcore/reduce_c.cpp:131] [SEGCORE][ReduceResultData][milvus] skip duplicated search result, count = 0


libgomp: Thread creation failed: Resource temporarily unavailable

libgomp: Thread creation failed: Resource temporarily unavailable

libgomp: Thread creation failed: Resource temporarily unavailable

libgomp: Thread creation failed: Resource temporarily unavailable

libgomp: Thread creation failed: Resource temporarily unavailable

libgomp: Thread creation failed: Resource temporarily unavailable
fatal error: unexpected signal during runtime execution
[signal SIGSEGV: segmentation violation code=0x80 addr=0x0 pc=0x7feffff1dfe6]

runtime stack:
runtime.throw(0x33c8dca, 0x2a)
        /usr/local/go/src/runtime/panic.go:1117 +0x72
runtime.sigpanic()
        /usr/local/go/src/runtime/signal_unix.go:718 +0x2e5

goroutine 111222364 [syscall]:
runtime.cgocall(0x1c64759, 0xc071946ce8, 0xc05c4e24c8)
        /usr/local/go/src/runtime/cgocall.go:154 +0x5b fp=0xc071946cb8 sp=0xc071946c80 pc=0x7c933b
github.com/milvus-io/milvus/internal/querynode._Cfunc_Search(0x7fa56fd9cc50, 0x7f9614fdc2d0, 0x7f9b8f394eb0, 0x602624c299c0003, 0xc05c4e24c8, 0x601751d92000001, 0x37200000000, 0x0)
        _cgo_gotypes.go:727 +0x51 fp=0xc071946ce8 sp=0xc071946cb8 pc=0x1c371b1
github.com/milvus-io/milvus/internal/querynode.(*Segment).search.func1(0xc021df8410, 0xc039777c08, 0x7f9b8f394eb0, 0x602624c299c0003, 0xc05c4e24c8, 0xc05c4e24c0, 0x0)
        /go/src/github.com/milvus-io/milvus/internal/querynode/segment.go:315 +0x125 fp=0xc071946d50 sp=0xc071946ce8 pc=0x1c51485
github.com/milvus-io/milvus/internal/querynode.(*Segment).search(0xc021df8410, 0xc039777c08, 0xc039777c20, 0x1, 0x1, 0xc041d31f28, 0x1, 0x1, 0x0, 0x0, ...)
        /go/src/github.com/milvus-io/milvus/internal/querynode/segment.go:315 +0x3b7 fp=0xc071946eb0 sp=0xc071946d50 pc=0x1c48137
github.com/milvus-io/milvus/internal/querynode.(*historical).search.func1(0xc07292fb50, 0xc00049eff0, 0x601751d92000001, 0xc01dfa4b80, 0xc039777c08, 0xc039777c20, 0x1, 0x1, 0x602624c299c0003, 0xc06cfc41b0, ...)
        /go/src/github.com/milvus-io/milvus/internal/querynode/historical.go:180 +0x19c fp=0xc071946f80 sp=0xc071946eb0 pc=0x1c4b9fc
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:1371 +0x1 fp=0xc071946f88 sp=0xc071946f80 pc=0x838a01
created by github.com/milvus-io/milvus/internal/querynode.(*historical).search
        /go/src/github.com/milvus-io/milvus/internal/querynode/historical.go:169 +0x5d8

goroutine 1 [chan receive, 63 minutes]:
github.com/milvus-io/milvus/cmd/roles.(*MilvusRoles).Run(0xc000a87ee0, 0x1, 0x0, 0x0)
        /go/src/github.com/milvus-io/milvus/cmd/roles/roles.go:495 +0x438
main.main()
        /go/src/github.com/milvus-io/milvus/cmd/main.go:296 +0x692

goroutine 373 [IO wait]:
internal/poll.runtime_pollWait(0x7fefb074e0f0, 0x72, 0xffffffffffffffff)
        /usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc0006dcb98, 0x72, 0x8000, 0x8000, 0xffffffffffffffff)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc0006dcb80, 0xc0002d0000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
        /usr/local/go/src/internal/poll/fd_unix.go:166 +0x1d5
net.(*netFD).Read(0xc0006dcb80, 0xc0002d0000, 0x8000, 0x8000, 0xcbcb7e, 0x800010601, 0x0)
        /usr/local/go/src/net/fd_posix.go:55 +0x4f
net.(*conn).Read(0xc000728860, 0xc0002d0000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
        /usr/local/go/src/net/net.go:183 +0x91
bufio.(*Reader).Read(0xc000aa78c0, 0xc0002c2038, 0x9, 0x9, 0x14, 0x7ff001cd5378, 0x18)
        /usr/local/go/src/bufio/bufio.go:227 +0x222
io.ReadAtLeast(0x36360a0, 0xc000aa78c0, 0xc0002c2038, 0x9, 0x9, 0x9, 0xc00069dce8, 0x8210f3, 0x5a3fc9a7ed7880)
        /usr/local/go/src/io/io.go:328 +0x87
io.ReadFull(...)
        /usr/local/go/src/io/io.go:347
golang.org/x/net/http2.readFrameHeader(0xc0002c2038, 0x9, 0x9, 0x36360a0, 0xc000aa78c0, 0x0, 0x0, 0xc0955662624ce3f6, 0x3719c67ee25)
        /go/pkg/mod/golang.org/x/net@v0.0.0-20210805182204-aaa1db679c0d/http2/frame.go:237 +0x89
golang.org/x/net/http2.(*Framer).ReadFrame(0xc0002c2000, 0xc0716a6f00, 0xc0716a6f00, 0x0, 0x0)
        /go/pkg/mod/golang.org/x/net@v0.0.0-20210805182204-aaa1db679c0d/http2/frame.go:492 +0xa5
google.golang.org/grpc/internal/transport.(*http2Client).reader(0xc0006383c0)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/http2_client.go:1347 +0x1a5
created by google.golang.org/grpc/internal/transport.newHTTP2Client
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/http2_client.go:346 +0xdd1

goroutine 210 [select]:
github.com/milvus-io/milvus/internal/util/rocksmq/server/rocksmq.(*retentionInfo).retention(0xc00036f420, 0x0, 0x0)
        /go/src/github.com/milvus-io/milvus/internal/util/rocksmq/server/rocksmq/rocksmq_retention.go:94 +0x139
created by github.com/milvus-io/milvus/internal/util/rocksmq/server/rocksmq.(*retentionInfo).startRetentionInfo
        /go/src/github.com/milvus-io/milvus/internal/util/rocksmq/server/rocksmq/rocksmq_retention.go:83 +0x5c

goroutine 211 [sleep, 3 minutes]:
time.Sleep(0x45d964b800)
        /usr/local/go/src/runtime/time.go:193 +0xd2
github.com/milvus-io/milvus/internal/util/rocksmq/server/rocksmq.NewRocksMQ.func1(0xc00049e180, 0xc00074f900)
        /go/src/github.com/milvus-io/milvus/internal/util/rocksmq/server/rocksmq/rocksmq_impl.go:227 +0x78
created by github.com/milvus-io/milvus/internal/util/rocksmq/server/rocksmq.NewRocksMQ
        /go/src/github.com/milvus-io/milvus/internal/util/rocksmq/server/rocksmq/rocksmq_impl.go:225 +0x628

goroutine 212 [select]:
github.com/uber/jaeger-client-go.(*RemotelyControlledSampler).pollControllerWithTicker(0xc00018bee0, 0xc000300050)
        /go/pkg/mod/github.com/uber/jaeger-client-go@v2.25.0+incompatible/sampler_remote.go:144 +0xa5
github.com/uber/jaeger-client-go.(*RemotelyControlledSampler).pollController(0xc00018bee0)
        /go/pkg/mod/github.com/uber/jaeger-client-go@v2.25.0+incompatible/sampler_remote.go:139 +0x73
created by github.com/uber/jaeger-client-go.NewRemotelyControlledSampler
        /go/pkg/mod/github.com/uber/jaeger-client-go@v2.25.0+incompatible/sampler_remote.go:86 +0x125

goroutine 215 [select]:
github.com/uber/jaeger-client-go/utils.(*reconnectingUDPConn).reconnectLoop(0xc00036f570, 0x6fc23ac00)
        /go/pkg/mod/github.com/uber/jaeger-client-go@v2.25.0+incompatible/utils/reconnecting_udp_conn.go:70 +0xc8
created by github.com/uber/jaeger-client-go/utils.newReconnectingUDPConn
        /go/pkg/mod/github.com/uber/jaeger-client-go@v2.25.0+incompatible/utils/reconnecting_udp_conn.go:60 +0x10c

goroutine 216 [select]:
github.com/uber/jaeger-client-go.(*remoteReporter).processQueue(0xc000655d40)
        /go/pkg/mod/github.com/uber/jaeger-client-go@v2.25.0+incompatible/reporter.go:296 +0xfe
created by github.com/uber/jaeger-client-go.NewRemoteReporter
        /go/pkg/mod/github.com/uber/jaeger-client-go@v2.25.0+incompatible/reporter.go:237 +0x1a5

goroutine 6931 [semacquire]:
sync.runtime_Semacquire(0xc00d964658)
        /usr/local/go/src/runtime/sema.go:56 +0x45
sync.(*WaitGroup).Wait(0xc00d964650)
        /usr/local/go/src/sync/waitgroup.go:130 +0x65
github.com/milvus-io/milvus/internal/msgstream.(*MqTtMsgStream).bufMsgPackToChannel(0xc001ccd320)
        /go/src/github.com/milvus-io/milvus/internal/msgstream/mq_msgstream.go:803 +0x20b
created by github.com/milvus-io/milvus/internal/msgstream.(*MqTtMsgStream).Start
        /go/src/github.com/milvus-io/milvus/internal/msgstream/mq_msgstream.go:751 +0x6e

goroutine 31509 [select, 63 minutes]:
google.golang.org/grpc.newClientStreamWithParams.func4(0xc000136000, 0xc00f553560, 0x3669478, 0xc00fe1cde0)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/stream.go:363 +0xaa
created by google.golang.org/grpc.newClientStreamWithParams
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/stream.go:362 +0x965

goroutine 218 [select, 63 minutes]:
google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc000866600)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/balancer_conn_wrappers.go:69 +0xac
created by google.golang.org/grpc.newCCBalancerWrapper
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/balancer_conn_wrappers.go:60 +0x172

goroutine 219 [chan receive, 63 minutes]:
google.golang.org/grpc.(*addrConn).resetTransport(0xc0004df8c0)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/clientconn.go:1214 +0x465
created by google.golang.org/grpc.(*addrConn).connect
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/clientconn.go:844 +0x12a

goroutine 14965 [select, 63 minutes]:
github.com/milvus-io/milvus/internal/datanode.(*channelEventManager).Run.func1(0xc0072a54c0)
        /go/src/github.com/milvus-io/milvus/internal/datanode/event_manager.go:66 +0xaa
created by github.com/milvus-io/milvus/internal/datanode.(*channelEventManager).Run
        /go/src/github.com/milvus-io/milvus/internal/datanode/event_manager.go:64 +0x3f

goroutine 221 [IO wait, 63 minutes]:
internal/poll.runtime_pollWait(0x7feffbea6cb8, 0x72, 0x0)
        /usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc000a18018, 0x72, 0x0, 0x0, 0x337a923)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Accept(0xc000a18000, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /usr/local/go/src/internal/poll/fd_unix.go:401 +0x212
net.(*netFD).accept(0xc000a18000, 0x37e11d600, 0x0, 0x0)
        /usr/local/go/src/net/fd_unix.go:172 +0x45
net.(*TCPListener).accept(0xc000642060, 0xc000871c28, 0xc000871c30, 0x28)
        /usr/local/go/src/net/tcpsock_posix.go:139 +0x32
net.(*TCPListener).Accept(0xc000642060, 0x3418880, 0xc000b86000, 0x3662e00, 0xc000642060)
        /usr/local/go/src/net/tcpsock.go:261 +0x65
google.golang.org/grpc.(*Server).Serve(0xc000b86000, 0x3662e00, 0xc000642060, 0x0, 0x0)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/server.go:786 +0x27f
github.com/milvus-io/milvus/internal/distributed/rootcoord.(*Server).startGrpcLoop(0xc0007e6b00, 0xcf6c)
        /go/src/github.com/milvus-io/milvus/internal/distributed/rootcoord/service.go:263 +0x7e5
created by github.com/milvus-io/milvus/internal/distributed/rootcoord.(*Server).startGrpc
        /go/src/github.com/milvus-io/milvus/internal/distributed/rootcoord/service.go:224 +0x6a

goroutine 73 [IO wait]:
internal/poll.runtime_pollWait(0x7feffbea6bd0, 0x72, 0xffffffffffffffff)
        /usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc0008fc118, 0x72, 0x8000, 0x8000, 0xffffffffffffffff)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc0008fc100, 0xc0005dc000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
        /usr/local/go/src/internal/poll/fd_unix.go:166 +0x1d5
net.(*netFD).Read(0xc0008fc100, 0xc0005dc000, 0x8000, 0x8000, 0xcbcb7e, 0x800010601, 0x0)
        /usr/local/go/src/net/fd_posix.go:55 +0x4f
net.(*conn).Read(0xc0005b8060, 0xc0005dc000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
        /usr/local/go/src/net/net.go:183 +0x91
bufio.(*Reader).Read(0xc0005c83c0, 0xc0007f6498, 0x9, 0x9, 0x14, 0x7ff001cda638, 0x18)
        /usr/local/go/src/bufio/bufio.go:227 +0x222
io.ReadAtLeast(0x36360a0, 0xc0005c83c0, 0xc0007f6498, 0x9, 0x9, 0x9, 0xc00069bce8, 0x8210f3, 0x5a3fcb07916a47)
        /usr/local/go/src/io/io.go:328 +0x87
io.ReadFull(...)
        /usr/local/go/src/io/io.go:347
golang.org/x/net/http2.readFrameHeader(0xc0007f6498, 0x9, 0x9, 0x36360a0, 0xc0005c83c0, 0x0, 0x0, 0xc0955663dc5b69f8, 0x372fc173028)
        /go/pkg/mod/golang.org/x/net@v0.0.0-20210805182204-aaa1db679c0d/http2/frame.go:237 +0x89
golang.org/x/net/http2.(*Framer).ReadFrame(0xc0007f6460, 0xc069f944e0, 0xc069f944e0, 0x0, 0x0)
        /go/pkg/mod/golang.org/x/net@v0.0.0-20210805182204-aaa1db679c0d/http2/frame.go:492 +0xa5
google.golang.org/grpc/internal/transport.(*http2Client).reader(0xc0005f4000)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/http2_client.go:1347 +0x1a5
created by google.golang.org/grpc/internal/transport.newHTTP2Client
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/http2_client.go:346 +0xdd1

goroutine 261 [select, 63 minutes]:
github.com/milvus-io/milvus/internal/indexnode.(*TaskScheduler).indexBuildLoop(0xc00020a960)
        /go/src/github.com/milvus-io/milvus/internal/indexnode/task_scheduler.go:285 +0x133
created by github.com/milvus-io/milvus/internal/indexnode.(*TaskScheduler).Start
        /go/src/github.com/milvus-io/milvus/internal/indexnode/task_scheduler.go:309 +0x5c

goroutine 74 [select]:
google.golang.org/grpc/internal/transport.(*controlBuffer).get(0xc0005da0a0, 0x1, 0x0, 0x0, 0x0, 0x0)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/controlbuf.go:407 +0xff
google.golang.org/grpc/internal/transport.(*loopyWriter).run(0xc0005c8480, 0x0, 0x0)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/controlbuf.go:527 +0x1dd
google.golang.org/grpc/internal/transport.newHTTP2Client.func3(0xc0005f4000)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/http2_client.go:396 +0x7b
created by google.golang.org/grpc/internal/transport.newHTTP2Client
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/http2_client.go:394 +0x12ae

goroutine 9966 [select]:
github.com/milvus-io/milvus/internal/util/flowgraph.(*nodeCtx).collectInputMessages(0xc00daf49a0)
        /go/src/github.com/milvus-io/milvus/internal/util/flowgraph/node.go:157 +0x130
github.com/milvus-io/milvus/internal/util/flowgraph.(*nodeCtx).work(0xc00daf49a0)
        /go/src/github.com/milvus-io/milvus/internal/util/flowgraph/node.go:96 +0x556
created by github.com/milvus-io/milvus/internal/util/flowgraph.(*nodeCtx).Start
        /go/src/github.com/milvus-io/milvus/internal/util/flowgraph/node.go:68 +0x55

goroutine 4570 [select, 63 minutes]:
google.golang.org/grpc.newClientStreamWithParams.func4(0xc00018ea80, 0xc003872a20, 0x3669478, 0xc0039f69f0)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/stream.go:363 +0xaa
created by google.golang.org/grpc.newClientStreamWithParams
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/stream.go:362 +0x965

goroutine 185 [select, 63 minutes]:
google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc000a2c240)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/balancer_conn_wrappers.go:69 +0xac
created by google.golang.org/grpc.newCCBalancerWrapper
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/balancer_conn_wrappers.go:60 +0x172

goroutine 186 [chan receive, 63 minutes]:
google.golang.org/grpc.(*addrConn).resetTransport(0xc0004de000)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/clientconn.go:1214 +0x465
created by google.golang.org/grpc.(*addrConn).connect
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/clientconn.go:844 +0x12a

goroutine 578 [select]:
github.com/milvus-io/milvus/internal/rootcoord.(*Core).startTimeTickLoop(0xc000625340)
        /go/src/github.com/milvus-io/milvus/internal/rootcoord/root_coord.go:281 +0xf7
created by github.com/milvus-io/milvus/internal/rootcoord.(*Core).Start.func1
        /go/src/github.com/milvus-io/milvus/internal/rootcoord/root_coord.go:1202 +0x28f

goroutine 188 [IO wait, 1 minutes]:
internal/poll.runtime_pollWait(0x7feffbea6ae8, 0x72, 0x0)
        /usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc00024f818, 0x72, 0x0, 0x0, 0x337a923)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Accept(0xc00024f800, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /usr/local/go/src/internal/poll/fd_unix.go:401 +0x212
net.(*netFD).accept(0xc00024f800, 0x37e11d600, 0x0, 0x0)
        /usr/local/go/src/net/fd_unix.go:172 +0x45
net.(*TCPListener).accept(0xc00000f020, 0xc0006f5930, 0xc0006f5938, 0x28)
        /usr/local/go/src/net/tcpsock_posix.go:139 +0x32
net.(*TCPListener).Accept(0xc00000f020, 0x3418880, 0xc000b86380, 0x3662e00, 0xc00000f020)
        /usr/local/go/src/net/tcpsock.go:261 +0x65
google.golang.org/grpc.(*Server).Serve(0xc000b86380, 0x3662e00, 0xc00000f020, 0x0, 0x0)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/server.go:786 +0x27f
github.com/milvus-io/milvus/internal/distributed/proxy.(*Server).startGrpcLoop(0xc0006d00c0, 0x4c4a)
        /go/src/github.com/milvus-io/milvus/internal/distributed/proxy/service.go:162 +0xccc
created by github.com/milvus-io/milvus/internal/distributed/proxy.(*Server).init
        /go/src/github.com/milvus-io/milvus/internal/distributed/proxy/service.go:215 +0x80f

goroutine 246 [IO wait]:
internal/poll.runtime_pollWait(0x7feffbea6a00, 0x72, 0xffffffffffffffff)
        /usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc0006dc118, 0x72, 0x8000, 0x8000, 0xffffffffffffffff)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc0006dc100, 0xc0009f6000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
        /usr/local/go/src/internal/poll/fd_unix.go:166 +0x1d5
net.(*netFD).Read(0xc0006dc100, 0xc0009f6000, 0x8000, 0x8000, 0xcbcb7e, 0x800010601, 0x0)
        /usr/local/go/src/net/fd_posix.go:55 +0x4f
net.(*conn).Read(0xc0003b6090, 0xc0009f6000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
        /usr/local/go/src/net/net.go:183 +0x91
bufio.(*Reader).Read(0xc000696120, 0xc0007f6578, 0x9, 0x9, 0x14, 0x7ff001cdeae8, 0x18)
        /usr/local/go/src/bufio/bufio.go:227 +0x222
io.ReadAtLeast(0x36360a0, 0xc000696120, 0xc0007f6578, 0x9, 0x9, 0x9, 0xc00089ace8, 0x8210f3, 0x5a3fca0d6c67b7)
        /usr/local/go/src/io/io.go:328 +0x87
io.ReadFull(...)
        /usr/local/go/src/io/io.go:347
golang.org/x/net/http2.readFrameHeader(0xc0007f6578, 0x9, 0x9, 0x36360a0, 0xc000696120, 0x0, 0x0, 0xc0955662cfca958c, 0x372011b33fa)
        /go/pkg/mod/golang.org/x/net@v0.0.0-20210805182204-aaa1db679c0d/http2/frame.go:237 +0x89
golang.org/x/net/http2.(*Framer).ReadFrame(0xc0007f6540, 0xc072c80df8, 0xc072c80df8, 0x0, 0x0)
        /go/pkg/mod/golang.org/x/net@v0.0.0-20210805182204-aaa1db679c0d/http2/frame.go:492 +0xa5
google.golang.org/grpc/internal/transport.(*http2Client).reader(0xc0005f41e0)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/http2_client.go:1347 +0x1a5
created by google.golang.org/grpc/internal/transport.newHTTP2Client
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/http2_client.go:346 +0xdd1

goroutine 327 [IO wait]:
internal/poll.runtime_pollWait(0x7feffbea6660, 0x72, 0xffffffffffffffff)
        /usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc0006dc198, 0x72, 0x8000, 0x8000, 0xffffffffffffffff)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc0006dc180, 0xc000b8c000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
        /usr/local/go/src/internal/poll/fd_unix.go:166 +0x1d5
net.(*netFD).Read(0xc0006dc180, 0xc000b8c000, 0x8000, 0x8000, 0xcbcb7e, 0x800010601, 0x0)
        /usr/local/go/src/net/fd_posix.go:55 +0x4f
net.(*conn).Read(0xc000728048, 0xc000b8c000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
        /usr/local/go/src/net/net.go:183 +0x91
bufio.(*Reader).Read(0xc000654300, 0xc0007f6118, 0x9, 0x9, 0x14, 0x7ff001cd5378, 0x18)
        /usr/local/go/src/bufio/bufio.go:227 +0x222
io.ReadAtLeast(0x36360a0, 0xc000654300, 0xc0007f6118, 0x9, 0x9, 0x9, 0xc0000dbce8, 0x8210f3, 0x5a3fc9a7ed5bc3)
        /usr/local/go/src/io/io.go:328 +0x87
io.ReadFull(...)
        /usr/local/go/src/io/io.go:347
golang.org/x/net/http2.readFrameHeader(0xc0007f6118, 0x9, 0x9, 0x36360a0, 0xc000654300, 0x0, 0x0, 0xc0955662624b1347, 0x3719c661d78)
        /go/pkg/mod/golang.org/x/net@v0.0.0-20210805182204-aaa1db679c0d/http2/frame.go:237 +0x89
golang.org/x/net/http2.(*Framer).ReadFrame(0xc0007f60e0, 0xc0716a6ee8, 0xc0716a6ee8, 0x0, 0x0)
        /go/pkg/mod/golang.org/x/net@v0.0.0-20210805182204-aaa1db679c0d/http2/frame.go:492 +0xa5
google.golang.org/grpc/internal/transport.(*http2Client).reader(0xc0005f45a0)
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/http2_client.go:1347 +0x1a5
created by google.golang.org/grpc/internal/transport.newHTTP2Client
        /go/pkg/mod/google.golang.org/grpc@v1.38.0/internal/transport/http2_client.go:346 +0xdd1

goroutine 413 [IO wait, 63 minutes]:
internal/poll.runtime_pollWait(0x7fefb074e1d8, 0x72, 0x0)
        /usr/local/go/src/runtime/netpoll.go:222 +0x55
internal/poll.(*pollDesc).wait(0xc000122598, 0x72, 0x0, 0x0, 0x337a923)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
        /usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Accept(0xc000122580, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /usr/local/go/src/internal/poll/fd_unix.go:401 +0x212
net.(*netFD).accept(0xc000122580, 0x30, 0x30, 0x7ff001cc4878)
        /usr/local/go/src/net/fd_unix.go:172 +0x45
net.(*TCPListener).accept(0xc000556078, 0xc0000d8d68, 0x7d3eb8, 0x30)
        /usr/local/go/src/net/tcpsock_posix.go:139 +0x32
net.(*TCPListener).Accept(0xc000556078, 0x3227ba0, 0xc0005ba3c0, 0x30b3920, 0x47354f0)
        /usr/local/go/src/net/tcpsock.go:261 +0x65
corrupted double-linked list
```

**Comment by yanliang567** [comment]
@sunby could you please help on this issue?

/assign @sunby 
/unassign

**Comment by karimBenzema12** [comment]
I am having the same problem. Will this issue be solved in version 2.0.3?
